addons/im_livechat/static/src/embed/common/livechat_service.js

```diff
@@ -64,41 +30,26 @@ export class LivechatService {
     /**
      * @param {import("@web/env").OdooEnv} env
      * @param {{
-     * bus_service: ReturnType<typeof import("@bus/services/bus_service").busService.start>,
      * "mail.store": import("@mail/core/common/store_service").Store
      * }} services
      */
     setup(env, services) {
         this.env = env;
-        this.busService = services.bus_service;
         this.notificationService = services.notification;
         this.store = services["mail.store"];
     }
 
     async initialize() {
-        const data =
-            this.options?.init ??
-            (await rpc("/im_livechat/init", {
-                channel_id: this.options.channel_id,
-            }));
-        this.available = data.available_for_me;
-        this.rule = this.store.LivechatRule.insert(data.rule);
-        this.store.insert(data.storeData);
-        if (this.options?.force_thread) {
-            this.state = SESSION_STATE.PERSISTED;
-            this.thread = this.store.Thread.insert(this.options.force_thread);
-            this._saveLivechatState();
-        }
-        if (this.savedState) {
-            this.state = this.savedState.persisted
-                ? SESSION_STATE.PERSISTED
-                : SESSION_STATE.CREATED;
-        }
-        if (this.state === SESSION_STATE.PERSISTED) {
-            await this.busService.addChannel(`mail.guest_${this.guestToken}`);
+        this.store.fetchStoreData("init_livechat", this.options.channel_id, { readonly: false });
+        if (this.options.test_chatbot_channel_id) {
+            await this.store.fetchStoreData("discuss.channel", [
+                this.options.test_chatbot_channel_id,
+            ]);
+            this.store.Thread.get({
+                id: this.options.test_chatbot_channel_id,
+                model: "discuss.channel",
+            }).openChatWindow();
```

Comment:
Note: should be improved to be set from python, probably in a similar way than chat requests. Keeping things simple to go ahead with this PR.